src/module/vue/components/foe-sheet.vue

```diff
@@ -126,7 +127,7 @@ function clearProgress() {
 }
 
 function markProgress() {
-  const increment = CONFIG.IRONSWORN.RankIncrements[foe.value?.data.rank]
+  const increment = RANK_INCREMENTS[foe.value?.data.rank]
   const newValue = Math.min(foe.value?.data.current + increment, 40)
```

Comment:
worth making the 40 a reference to a constant throughout the module rather than a "magic number"? mainly to make the purpose of the code really clear at a glance. perhaps something like the following:

```typescript
const TICKS_PER_BOX = 4
const PROGRESS_MAX_BOXES = 10
const PROGRESS_MAX_TICKS = PROGRESS_MAX_BOXES * TICKS_PER_BOX
```